src/main/java/commands/AddCommand.java

```diff
@@ -0,0 +1,37 @@
+package commands;
+
+import storage.Storage;
+import tasklist.TaskList;
+import tasks.DukeTask;
+import ui.Ui;
```

Comment:
Your imported classes are listed explicitly. You did this for other classes as well. Nice one!